src/NodeDynamics/FourthOrderEquation.jl

```diff
@@ -6,25 +6,32 @@
 FourthEq(H, P, D, Ω, E_f, T_d_dash ,T_q_dash ,X_q_dash ,X_d_dash,X_d, X_q)
 ```
 
-A node type that applies the 4th-order synchronous machine model
-with frequency/angle and voltage dynamics.
+A node type that applies the 4th-order synchronous machine model with frequency/angle and voltage dynamics,
+which is implemented according to P. Sauer, "Power System Dynamics and Stability".
+For an illustration of a synchronous machine schematic see P. Sauer, Fig. 3.1 on p. 25.
+
+Usually the swing equation (`SwingEq`) is used for short time periods to analyze the transient behavior
+of generators in a power grid, the so-called first swing. The 4th-order model  also takes the back reaction
+of the power flow onto the voltage into account. This has the effect that the angle of the voltage as seen by the power grid,
+and the angle of the rotating mass are no longer the same but become dynamically coupled.
 
 Additionally to ``u``, it has the internal dynamic variables
-* ``ω`` representing the frequency of the rotator relative to the grid frequency ``\Omega``, i.e. the real frequency ``\omega_r`` of the rotator is given as ``\omega_r = \Omega + \omega`` and
-* ``θ`` representing the relative angle of the rotor with respect to the voltage angle ``\phi``.
+* ``ω`` representing the frequency of the rotator relative to the grid frequency ``Ω``, i.e. the real frequency ``ω_r`` of the rotator is given as ``\omega_r = \Omega + \omega`` and
+* ``θ`` representing the relative angle of the rotor with respect to the voltage angle ``ϕ``.
 
 # Keyword Arguments
-- `H`: inertia
-- `P`: active (real) power output
-- `D`: damping coefficient
+- `H`: shaft inertia constant (given in [s]), defined according to P. Sauer, p. 33, eq. (3.60)
+- `P`: active (real) power output, also called the mechanical torque applied to the shaft, given in [pu]
+- `D`: damping coefficient (given in [s], see P. Sauer, eq. (5.156) where the damping torque is equal `Dω`)
 - `Ω`: rated frequency of the power grid, often 50Hz
-- `Ω_H`: normalized `Ω` according to ``Ω_H = (Ω * 2\pi) / H``
-- `T_d_dash`: time constant of d-axis
-- `T_q_dash`: time constant of q-axis
-- `X_d_dash`: transient reactance of d-axis
-- `X_q_dash`: transient reactance of q-axis
-- `X_d`: reactance of d-axis
-- `X_d`: reactance of q-axis
+- `Ω_H`: normalized `Ω` (given in [``s^{-2}``]) according to `Ω_H = (Ω⋅2\pi) / H`
```

Comment:
This is not a Keyword argument, right? It's just a temporary variable computed in the code. It should be removed here. And I don't actually think there should be any further documentation of this variable except what is already directly in the code.